CHANGELOG.md

```diff
@@ -42,6 +42,10 @@ This version brings a breaking change with the move to use the [Zarr](https://za
 - Bug fix for allowing a period to be specified when this cannot be inferred from the input data [PR #899](https://github.com/openghg/openghg/pull/899)
 
 
+
+### Fixed
```

Comment:
One note is that now we're released 0.8.0, this should be moved to the Unreleased part of the CHANGELOG.md file (i.e. between current lines 8 and 10)

Reply:
Yes, Nice catch. Thanks for pointing it out.

Reply:
done